install-dynamodb-local.sh

```diff
@@ -0,0 +1,23 @@
+
+#!/bin/bash
+
+
+echo "Installing Java"
+brew install openjdk
+
+echo "java is installed sucessfully.... :) "
+echo " "
+echo " "
```

Comment:
I think you can add `\n` for newline, if you wanted to.

```suggestion
echo "java is installed sucessfully.... :) \n\n"
```